src/pages/Login/styled.scss

```diff
@@ -0,0 +1,74 @@
+.login-page {
+  .title-login {
+    padding: 50px;
+  }
+  .label-input {
+    font-size: 20px;
+    font-weight: 700;
+    color: #777e90;
+    margin-bottom: 12px;
+  }
+  .clock-login {
+    width: 368px;
+    height: 36px;
+    margin: 15px auto 10px;
+    border-radius: 100px;
+    background: rgba(88, 189, 125, 0.1);
+    display: flex;
+    align-items: center;
+  }
+  .clock-icon {
+    width: 36px;
+    height: 36px;
+    min-width: 36px;
+    background: #58bd7d;
+    border-radius: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  .clock-url {
+    padding-left: 44px;
+    font-weight: 500;
+  }
+  .form-login {
+    max-width: 690px;
+    margin: auto;
+    button {
+      padding: 25px;
+    }
+  }
+  .full-width {
+    width: 100%;
+  }
+  input {
```

Comment:
this will effect other input, please wrap it with other class or put className to this input